### DISCUSSION TITLE
Modernize README

### DISCUSSION THREAD

**Discussion by LilianBoulard** [skrub-data/skrub#272] [Ideas]
I really like the README of [CleverCSV](https://github.com/alan-turing-institute/CleverCSV), and I think it would be great if ours was a bit more like it :)
I know the usage is very different, but still I think there are some good ideas to extract !

**Comment by alexis-cvetkov** [comment]
I agree that the readme could be improved ! Here are some suggestions:
- A section "Why use dirty-cat ?" to better motivate the package, with an image of a table with a dirty categorical column (e.g. from the video) and maybe a picture showing the improvement in performance with one-hot encoding
- A section "How to use" with a simple code example
- A section "Encoders" that quickly list the encoders and their features

**Comment by GaelVaroquaux** [comment]
I worry that this is in fact duplicating the docs.

If we adds all this information there, why not in the docs?

If we duplicate the information, how do we make sure that informations stay up to date (hint: I worry that they won't).

I'd rather that we invest time in improving our docs

**Comment by LilianBoulard** [comment]
I think they have different objectives: IMO the readme should be a short extract of what the library has to offer, without going in the details; those should be on the website and in the examples.

The readme should be the frontpage for the newcomers: people stumbling upon the repo (or the PyPI for that matter) and not knowing what the library does. Currently, the readme doesn't convey this information very well.

I'd say that the information we'd like to put in the readme would at most be duplicating the information we have in the introductory video: a motivation for the issue we're trying to solve, the list of tools we implement to solve these (without getting into any detail), and a small benchmark to prove that these methods indeed work.
And to avoid duplicating this information (which could be on the website as well !), we can either integrate the readme or a part of it on the website (automatically, I mean), or put a link to it.

I agree with Alexis on the structure, though the "Encoders" section would be duplicating the info currently on main page of the website, so I think we should stay general in the readme !
